backend/app/main.py

```diff
@@ -19,47 +19,7 @@
     allow_headers=['*'],
 )
 
-
-def get_db():
```

Comment:
I think we'll need the `get_db` function pretty much everywhere. Is there somewhere you could stick it that'd be available to all routers?

Reply:
That is a good idea! I moved it to database.py and included it in the import on the router. For the other routers we can use a similar import:

`from ..database import SessionLocal, engine, get_db`